### ISSUE TITLE
bad-whitespace false positive with variable-length type hint keyword argument

### ISSUE THREAD

**Issue by jwkvam** [PyCQA/pylint#2206]
### Steps to reproduce
1. Create failing file:
Failing file:
```
from typing import Tuple
def hello(x: Tuple[int, ...] = (3,)):
    print(x)
```

Example passing file:
```
from typing import Tuple
def hello(x: Tuple[int, int] = (3,)):
    print(x)
```
2. Run: `pylint file.py`

### Current behavior
Pylint outputs:
```
C:  2, 0: No space allowed around keyword argument assignment
def hello(x: Tuple[int, ...] = (3,)):
                             ^ (bad-whitespace)
```

### Expected behavior
No error

### pylint --version output
```
pylint 1.9.2,
astroid 1.6.4
Python 3.6.5 | packaged by conda-forge | (default, Apr  6 2018, 13:44:09)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)]
```

**Comment by PCManticore** [comment]
Thanks for the report!

**Comment by brycepg** [comment]
Duplciate of #1992
